include/xwayland/selection.h

```diff
@@ -77,6 +77,6 @@ bool primary_selection_source_is_xwayland(
 void xwm_seat_handle_start_drag(struct wlr_xwm *xwm, struct wlr_drag *drag);
 
 void xwm_selection_init(struct wlr_xwm *xwm);
-void xwm_selection_finish(struct wlr_xwm *xwm);
+void xwm_selection_finish(struct wlr_xwm_selection *selection);
```

Comment:
Hm, this creates a discrepancy with `xwm_selection_init`. Either:

- Maybe just rename to `xwm_finish_selection`?
- Maybe take a `struct wlr_xwm_selection *` in `xwm_selection_init` too?

Reply:
I've made `xwm_selection_finish` take a `struct wlr_xwm_selection *` too now.